app/src/main/res/layout/share_close.xml

```diff
@@ -36,7 +37,7 @@
         android:paddingHorizontal="1dp"
         android:text="@string/share_header_2"
         android:textAppearance="@style/HeaderTextStyle"
-        android:textColor="@color/neutral_text"
+        android:textColor="?primaryText"
```

Comment:
This is probably my fault, but I have changed `?primaryText` to map to `fx_mobile_text_color_primary` for the light theme. As a result, this doesn't quite look right under light theme at the moment.

 I am waiting until after the beta cut to make more fo these changes since you can imagine we will effectively change the color of the text and icons everywhere. This will mean we should probably wait until after the beta cut on monday before landing this PR.